packages/dotcom-ui-header/src/components/ask-ft/askFtButton.tsx

```diff
@@ -2,19 +2,18 @@ import React from 'react'
 
 export interface AskFtButtonProps {
   id: string
-  className: string
```

Comment:
**thought:** This is considered a breaking change. We should release this as a new major version.

Reply:
As per my understanding it is not a breaking change, because `AskFtButton` component is used only internally in `dotcom-ui-header` and all places were updated accordingly.